tests/test_suite.html

```diff
@@ -404,6 +404,71 @@
             w6CssInjected.should.equal(true);
         });
 
+        // decisionAdapter
+        var w7DecisionAdapterCalled = false;
+        Mojito.options.decisionAdapter = function(testObject)
+        {
+            w7DecisionAdapterCalled = true;
+            return 0.8;
+        };
+
+        Mojito.addTest({
+            id: 'w7',
+            state: 'live',
+            name: 'unit_test_decision_adapter',
+            sampleRate: 1,
+            trigger: function (e) {
+                e.activate();
+            },
+            recipes: {
+                '0': {
+                    name: 'control'
+                },
+                '1': {
+                    name: 'treatment'
+                }
+            }
+        });
+
+        it('Mojito.options.decisionAdapter was called', function () {
+            w7DecisionAdapterCalled.should.equal(true);
+        });
+
+        var w8ControlChosenByDecisionAdapter = false;
+        Mojito.options.decisionAdapter = function(testObject)
```

Comment:
For both tests, could you please add the `decisionAdapter` to the testObject directly?

I think there are some tests earlier on that activate after these do and therefore may be influenced from setting decisionAdapters here.

Reply:
Fixed.